src/MUS6.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Directive variants
+  from: '@ingydotnet'
+  tags: directive
+  yaml: |
+    %YAML  1.1
+    ---
+  tree: |-
+    +STR
+     +DOC ---
+      =VAL :
+     -DOC
+    -STR
+  json: |
+    null
+
+- yaml: |
+    %YAML 1.1 #...
+    ---
+
+- yaml: |
+    %YAM 1.1
+    ---
+
+- yaml: |
+    %YAMLL 1.1
+    ---
+
+- yaml: |
+    %YAML———»1.1
+    ---
+
+- yaml: |
+    %YAML1.1
+    ---
+
+- yaml: |
+    %***
+    ---
+
+- yaml: |
+    %YAML 1.1 1.2
+    ---
+
+- yaml: |
+    %YAML 1.12345
+    ---
+
```

Comment:
These are all supposed to be valid?

Reply:
I wish they were errors but they all match https://yaml.org/spec/1.2.2/#rule-ns-reserved-directive 
and they don't match https://yaml.org/spec/1.2.2/#rule-ns-yaml-directive
and https://yaml.org/spec/1.2.2/#rule-l-directive says to match the former if the latter doesn't match.

This is something I'd like to "fix" in 1.3 so maybe we should remove these kind of things from the suite to not encourage frameworks to go one way, then reverse course later.

There's lot of annoying situations like this that were not well thought out.

Reply:
yeah, maybe some of these tests should be marked as `skip` for now, I wouldn't feel good implementing this.

Reply:
Fixed in 3c797dd

Moved the tests that should probably be errors to a file that gets skipped.

We can't currently skip individual tests, only files.